src/Components/RosTable/ProgressScoreBar.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Progress } from '@patternfly/react-core';
+import propTypes from 'prop-types';
+
+export const ProgressScoreBar = ({ valueScore, measureLocation }) =>  {
+    const calNumScore = Math.round(valueScore / 20);
+    const textLabel = `${ calNumScore }/5`;
+    const colorclass = (val) => {
+        switch (true) {
+            case (val === 5):
+                return 'green-400';
+            case (val === 4):
+                return 'green-100';
+            case (val === 3):
+                return 'gold-400';
+            case (val === 2):
+                return 'orange-300';
+            case (val <= 1):
+                return 'danger-200';
+        }
+    };
+
+    return (
+        <React.Fragment>
+            <Progress value={ calNumScore } min={0} max={5} label={ textLabel }
+                valueText={ textLabel }
+                className={ `progress-score-bar ${colorclass(calNumScore)}` }
+                measureLocation={ measureLocation } />
+        </React.Fragment>
+    );
+};
```

Comment:
@karelhala, 
Here you suggested to make use of `variant` but if you see attached screenshot from [comment-744601282](https://github.com/RedHatInsights/ros-frontend/pull/4#issuecomment-744601282).

I need to use different shades of these colours (e.g. dark danger or light green) than existing variant. Is there any way I can pass colour intensity number (100/200/300/400) along with these variants?

Reply:
Secondly, from server I am getting percentage value in value score.
After division by 20, I am considering round figure while showing a progress as well as text (i.e x out of 5). 
Example - result = 4.5 after division it will show 5/5 and 100% progress.

If I change code as per suggestion then there will be difference in progress (like for 95%, 98% and so on till 100%)  but text will show `5/5. It will be little confusing. Please let me know your thoughts on this.